### DISCUSSION TITLE
Improvement: Fix "title" error for missing block configurations

### DISCUSSION THREAD

**Discussion by kirkbushell** [area17/twill#861] [Q&A]
When you're setting up a block, and forget to configure it at config/twill.php - you get an error message that is quite unclear (it's taken me 4 or 5 times seeing this error message to actually understand and remember what is going on):

> Block cards does not exists or the mandatory property 'title' was not found on this block. If you are still using blocks on the twill.php file, please check if the block is present and properly configured.

A better error message, before this - would be to check that the configuration actually exists, and if not - throw an error (title check could be done after). Better yet, why array configuration? Why not a new Block object that could be registered with the system?

**Comment by ptrckvzn** [comment]
Hi @kirkbushell, I believe the Blocks are being refactored to be created/registered when a view exist in `resources/views/admin/blocks` and the block configuration array in `config/twill.php` will be deprecated. Until the documentation is updated, does the section under [Self-contained blocks](https://github.com/area17/twill/blob/2.x/CHANGELOG.md#added) helps understand what's going on?

In the issue you're having, adding the prop `@twillBlockTitle('Your title')` will probably fix your issue (without adding something to the config).

**Comment by ptrckvzn** [comment]
We'll address this in PR (#813) for the next release.
